core/schema/service.go

```diff
@@ -136,22 +145,63 @@ func (s *Service) Create(ctx context.Context, nsName string, schemaName string,
 func (s *Service) identifySchemaChange(ctx context.Context, request *changedetector.ChangeRequest) error {
 	endFunc := s.newrelic.StartGenericSegment(ctx, "Identify Schema Change")
 	defer endFunc()
+	schemaID, err := s.repo.GetSchemaID(ctx, request.NamespaceID, request.SchemaName)
+	if err != nil {
+		log.Printf("got error while getting schema ID from DB %s", err.Error())
+		return err
+	}
+	prevEvent, err := s.notificationEventRepo.GetByNameSpaceSchemaVersionAndSuccess(ctx, request.NamespaceID, schemaID, request.VersionID, true)
+	if err != nil && !errors.Is(err, pgx.ErrNoRows) {
+		log.Printf("got error while fetching previous notification event status for for namespace : %s, schema: %s, version: %d, %v", request.NamespaceID, request.SchemaName, request.Version, err)
+	}
+	if prevEvent.ID != "" {
```

Comment:
Need context: what is the meaning of != "" ?

Reply:
if there is some previousEvent then id will not be "" and we need to skip that event.